python/proton/_message.py

```diff
@@ -440,8 +465,8 @@ def _set_annotations(self, annotations):
     annotations = property(_get_annotations, _set_annotations, doc="""
     Message annotations as a dictionary of key/values. The AMQP 1.0
     specification restricts this dictionary to have keys that are either
-    :class:`symbol` or :class:`ulong` types. It is recommended that
-    the special ``dict`` subclass :class:`AnnotationDict` is used as it
+    :class:`symbol` or :class:`ulong` types. It is possible to use
+    the special ``dict`` subclass :class:`AnnotationDict` as it
```

Comment:
"It is possible to do X if you wish to achieve Y", or something like that.  "It is possible to do X as it does Y" doesn't work for me.  I think the idea is to use it if you want the behavior, not because of the behavior.

Reply:
Changed wording.